packages/xchain-litecoin/src/client.ts

```diff
@@ -323,6 +324,36 @@ class Client extends UTXOClient {
       auth: this.nodeAuth,
     })
   }
+
+  /**
+   * Transaction to THORChain inbound address.
+   *
+   * @param {DepositParams} params The transaction options.
+   * @returns {TxHash} The transaction hash.
+   *
+   * @throws {"halted chain"} Thrown if chain is halted.
+   * @throws {"halted trading"} Thrown if trading is halted.
+   */
+  async deposit({ walletIndex = 0, asset = AssetLTC, amount, memo }: DepositParams): Promise<TxHash> {
+    const inboundDetails = await getInboundDetails(asset.chain, this.network)
+
+    if (inboundDetails.haltedChain) {
+      throw new Error('halted chain')
```

Comment:
[suggestion] `Error('halted chain')` -> `Error(`Halted chain for ${assetToString(asset)}`)

Do this also in `deposit` methods of other packages.